src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -442,13 +448,37 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 			dropdownMenuActionViewItem.render(buttonDropdownContainer);
 			dropdownMenuActionViewItem.setActionContext(this._notebookParams.notebookUri);
 
+			let viewsDropdownContainer;
+			if (this._configurationService.getValue<boolean>('notebookViews.enabled')) {
+				let viewsContainer = document.createElement('li');
+				let viewsActionsProvider = new NotebookViewsOptions(viewsContainer, this.views, this.modelReady, this.notebookService, this.instantiationService);
+				let viewsButton = this.instantiationService.createInstance(AddCellAction, 'notebook.OpenViews', localize('views', "Views"), 'notebook-button masked-pseudo code');
+				viewsDropdownContainer = DOM.$('li.action-item');
+				viewsDropdownContainer.setAttribute('role', 'presentation');
+				let viewsDropdownMenuActionViewItem = new DropdownMenuActionViewItem(
+					viewsButton,
+					viewsActionsProvider,
+					this.contextMenuService,
+					undefined,
+					this._actionBar.actionRunner,
+					undefined,
+					'codicon notebook-button masked-pseudo masked-pseudo-after icon-dashboard-view dropdown-arrow',
+					localize('editor', "Editor"),
+					undefined
+				);
+				viewsDropdownMenuActionViewItem.render(viewsDropdownContainer);
+				viewsDropdownMenuActionViewItem.setActionContext(this._notebookParams.notebookUri);
+			}
+
 			this._actionBar.setContent([
 				{ element: buttonDropdownContainer },
 				{ action: this._runAllCellsAction },
 				{ element: Taskbar.createTaskbarSeparator() },
 				{ element: kernelContainer },
 				{ element: attachToContainer },
 				{ element: spacerElement },
+				{ element: Taskbar.createTaskbarSeparator() },
+				{ element: viewsDropdownContainer },
```

Comment:
It would be better to splice in the container if the configuration for that is enabled rather than inserting an undefined element when it isn't.